### PR TITLE
update aws cloudwatch scaler doc

### PR DESCRIPTION
**Pull request by fivesheep**
Signed-off-by: Xiayang Wu <xwu@rippling.com>

_Provide a description of what has been changed_

### Checklist

- [ ] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

update `content/docs/2.4/scalers/aws-cloudwatch.md` and `content/docs/2.5/scalers/aws-cloudwatch.md` for the two new parameters `metricUnit` and `metricEndTimeOffset` that are introduced by https://github.com/kedacore/keda/pull/2243

### CODE REVIEW

**Review by JorTurFer — CHANGES_REQUESTED** [review]
Thanks for your contribution!!
Could you undo the changes in v2.4? The changes that you have done will be merged with main branch and released in the version 2.5. Since v2.4 was released some time ago, it has no sense update it :)

**Review by JorTurFer — APPROVED** [review]
LGTM!

File: content/docs/2.5/scalers/aws-cloudwatch.md [hunk 1]
```diff
@@ -45,9 +49,10 @@ triggers:
...
-- `metricStatPeriod` - Which frequency is going to be used by the related query. Used to define **Period** ([official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html)). (Default: `300`, Optional)
+- `metricCollectionTime` - How long in the past (seconds) should the scaler check AWS Cloudwatch. Used to define **StartTime** ([official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricData.html)). The value of `metricCollectionTime` must be greater than the `metricStatPeriod`, providing a value which is a multiply of the `metricStatPeriod` can improve performance on fetching data from Cloudwatch. In pratice setting `metricCollectionTime` 2-to-3 times more than the `metricStatPeriod` value can make sure the scaler is able to get data points back from Cloudwatch, the scaler will always use the most up-to-date datapoint if more datapoints are returned. (Default: `300`, Optional)
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/aws-cloudwatch.md:52]: Can you also add `metricUnit` here please?

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

**Review by tomkerkhove — APPROVED** [review]

**Comment by tomkerkhove** [comment]
Thank you! Waiting for the reature to merge so we can close this.

**Pull request merged**